src/content/terraform/additional-sidebar-resources.json

```diff
@@ -0,0 +1,6 @@
+[
+	{
+		"title": "Terraform Registry",
+		"href": "https://registry.terraform.io/"
```

Comment:
In the design spec, this item is supposed to come _before_ other resource items.

However, given we'd already built this functionality, my intent here is to run this by design and see if it'd be fine to put the extra `Registry` item after other items, to avoid a refactor here.

> **Note**: 💭 If design strictly wants the item first, one approach that feels viable to me would be, instead of having optional `additional-sidebar-resources.json`, have an optional `sidebar-resources.json`. We'd then either use that authored set of resources if present, or fall back to a default set like we're currently generating. But we would not try to mix authored resources with fallback defaults.

Reply:
`additional-sidebar-resources.json` was a product-driven decision and. See this Slack thread for context: https://hashicorp.slack.com/archives/C03A8CW77FU/p1659577575329129. We should get design to update the Figma to reflect the decision that was made in that thread.